sdk/ai/Azure.AI.Projects/src/Custom/Agent/AgentsClient.Streaming.cs

```diff
@@ -163,6 +163,60 @@ public virtual CollectionResult<StreamingUpdate> CreateRunStreaming(string threa
         Response sendRequest() => CreateRunStreaming(threadId, createRunRequest.ToRequestContent(), context);
         return new StreamingUpdateCollection(sendRequest, cancellationToken);
     }
+    /// <summary> Submits outputs from tools as requested by tool calls in a stream. Stream updates that need submitted tool outputs will have a status of 'RunStatus.RequiresAction'. </summary>
+    /// <param name="threadId"> Identifier of the thread. </param>
+    /// <param name="runId"> Identifier of the run. </param>
+    /// <param name="toolOutputs"> A list of tools for which the outputs are being submitted. </param>
+    /// <param name="cancellationToken"> The cancellation token to use. </param>
+    /// <exception cref="ArgumentNullException"> <paramref name="threadId"/>, <paramref name="runId"/> or <paramref name="toolOutputs"/> is null. </exception>
+#pragma warning disable AZC0015 // Unexpected client method return type.
+    public virtual CollectionResult<StreamingUpdate> SubmitToolOutputsToStream(string threadId, string runId, IEnumerable<ToolOutput> toolOutputs, CancellationToken cancellationToken = default)
+#pragma warning restore AZC0015 // Unexpected client method return type.
+    {
+        Response sendRequest() => SubmitToolOutputsToRunInternal(threadId, runId, toolOutputs, true, cancellationToken);
```

Comment:
To get a streaming response instead of getting all the output at once, you need to set `BufferResponse` to `false`, as shown [here](https://github.com/Azure/azure-sdk-for-net/blob/3575d8d3466cf825fafe38eaea5f11cc287ad3e6/sdk/ai/Azure.AI.Projects/src/Custom/Agent/AgentsClient.Streaming.cs#L231).

Reply:
Fixed.